regress/lwgeom_regress_expected

```diff
@@ -21,3 +21,11 @@ BoundingDiagonal3|SRID=4326;LINESTRING(999999986991104 999999986991104,999999986
 BoundingDiagonal4|SRID=3857;LINESTRING(-1 -2 -8 2,1 2 3 9)
 BoundingDiagonal5|SRID=3857;LINESTRINGM(4 4 0,5 4 1)
 BoundingDiagonal6|SRID=3857;LINESTRINGM EMPTY
+ST_Azimuth_regular|2.3562
+ST_Azimuth_same_point|
+ERROR:  Operation on mixed SRID geometries
+ERROR:  Argument must be POINT geometries
+ST_Angle_4_pts|4.71238898038469
+ST_Angle_4_pts|0.785398163397448
+ST_Angle_3_pts|1.5707963267949
+ERROR:  Operation on mixed SRID geometries
```

Comment:
> I removed the empty test case because ST_Azimuth is failing it.
> (it returns another error than the one expected)
> it returns `ERROR:  getPoint2d_p: point offset out of range` 

Evidently your is_empty calls are not working.
Better fix that first.